easybuild/easyconfigs/p/pkg-config/pkg-config-0.29.2-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,36 @@
+# Modified by MKr (CSCS), 02/03/2020
+easyblock = 'ConfigureMake'
+
+name = 'pkg-config'
+version = '0.29.2'
+
+homepage = 'https://www.freedesktop.org/wiki/Software/pkg-config/'
+
+description = """
+ pkg-config is a helper tool used when compiling applications and libraries.
+ It helps you insert the correct compiler options on the command line so an
+ application can use gcc -o test test.c `pkg-config --libs --cflags glib-2.0`
+ for instance, rather than hard-coding values on where to find glib (or other
+ libraries).
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+sources = ['/apps/common/UES/easybuild/sources/p/pkg-config/pkg-config-0.29.2.tar.gz']
```

Comment:
Just for the record, absolute path is used here because there is currently an issue downloading this file from the tds node.